app/components/chart-published.tsx

```diff
@@ -292,82 +295,84 @@ const ChartPublishedInner = (props: ChartPublishInnerProps) => {
             </Box>
           )}
           <LoadingStateProvider>
-            <InteractiveFiltersProvider>
-              <Flex
-                sx={{
-                  justifyContent: meta.title[locale]
-                    ? "space-between"
-                    : "flex-end",
-                  gap: 2,
-                }}
-              >
-                {meta.title[locale] ? (
-                  <Title text={meta.title[locale]} />
+            <InteractiveFiltersProvider chartConfigs={state.chartConfigs}>
```

Comment:
I think now we have two nested `InteractiveFiltersProviders`, one in `ChartPublished` and one here, in `ChartPublishedInner`. Shouldn't there only be one?

Reply:
Nice catch.